doc.config.json

```diff
@@ -131,9 +131,66 @@
             ]
           },
           {
-            "title": "Nitric API",
+            "title": "JavaScript",
```

Comment:
I think Node.js may work better here, as the SDK isn't web compatible at the moment.